nixos/modules/tasks/filesystems/envfs.nix

```diff
@@ -31,11 +31,19 @@ in {
           etc.
         '';
       };
+
       package = lib.mkOption {
         type = lib.types.package;
-        description = lib.mdDoc "Which package to use for the envfs.";
         default = pkgs.envfs;
         defaultText = lib.literalExpression "pkgs.envfs";
+        description = lib.mdDoc "Which package to use for the envfs.";
+      };
```

Comment:
```suggestion
      fallbackPathPackage = lib.mkOption {
        type = lib.types.package;
        description = lib.mdDoc "package that contains fallback executables in case not other executable is found";
        default = TODO;
        description = lib.mdDoc "Which package to use for the envfs.";
      };
```

Reply:
and have a default value in `config = {};` or in the option.